src/tools/fuzzing.h

```diff
@@ -327,8 +327,10 @@ class TranslateToFuzzReader {
     options.push_back(type); // includes itself
     switch (type.getSingle()) {
       case Type::anyref:
+        if (wasm.features.hasExceptionHandling()) {
+          options.push_back(Type::exnref);
+        }
         options.push_back(Type::funcref);
```

Comment:
I assume we don't need to check for reference-types here because the entire function is only called in that case?

Reply:
I think the function will still be called, but never with `Type::anyref` unless reference types are enabled. This case requires special handling because the subtype needs more features than the supertype.

Reply:
I see, thanks!